core/templates/pages/topic-viewer-page/topic-viewer-page.directive.html

```diff
@@ -11,7 +11,7 @@
            translate="I18N_TOPIC_VIEWER_PLAY">
         </a>
       </li>
-      <li ng-class="{'topic-viewer-tabs-active': $ctrl.activeTab === 'practice'}">
+      <li ng-class="{'topic-viewer-tabs-active': $ctrl.activeTab === 'practice'}" ng-hide="!$ctrl.showTrainTab">
```

Comment:
Maybe use ng-show so you don't need the double negative?

Reply:
Done.